internal/featuresupport/features_olm_operators_test.go

```diff
@@ -264,8 +264,8 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 		},
 			Entry("on Vsphere", "4.14", common.X86CPUArchitecture, models.PlatformTypeVsphere, models.SupportLevelUnavailable),
 			Entry("on Nutanix", "4.14", common.X86CPUArchitecture, models.PlatformTypeNutanix, models.SupportLevelUnavailable),
-			Entry("on None", "4.14", common.X86CPUArchitecture, models.PlatformTypeNone, models.SupportLevelUnavailable),
```

Comment:
Why did you remove it instead if changing to supported.

Reply:
fixed